openstates/nv/bills.py

```diff
@@ -245,8 +245,18 @@ def scrape_actions(self, root, bill, actor):
                         action_type = atype
                         break
 
+
+                if "Committee on" in action:
+                    committees = re.findall("Committee on ([a-zA-Z, ]*)\.",action)
```

Comment:
That's what the literal period is for! I checked a bunch and they all had extra text, but the committee was always the end of a sentence.